janus_core/stats.py

```diff
@@ -0,0 +1,69 @@
+"""
+Module that reads the md stats output timeseries.
+"""
+
+import re
+
+from numpy import genfromtxt
+
+from janus_core.janus_types import PathLike
+
+
+class Stats:
```

Comment:
We should have methods to get names of contained stats. I realise this is in `labels`, but getting a zip of `labels` and `units` may also be handy.
Do we want `__getitem__` definitions to pull out rows? 
Do we want getters for contained stats?

Reply:
that will be nice but not something I have time to do @ElliottKasoar maybe at tomorrow's hackaton? the only issue I have is some of the fields are T* i doubt a getter will like that

Reply:
opened an issue about it.